File: IOSwiftUI/Backoffice.UI/src/screens/members/models/BOMemberListResponseModel.ts
import BaseResponseModel from "../../../common/models/BaseResponseModel";
import BOMemberModel from "./BOMemberModel";

class BOMemberListResponseModel extends BaseResponseModel {

    count: number;
    memberList: BOMemberModel[];

    constructor() {
        super();

        this.count = 0;
        this.memberList = [];
    }
}

export default BOMemberListResponseModel;

File: IOSwiftUI/Common/Messages/Members/BOMemberListResponseModel.cs
using System;
using System.Collections.Generic;
using IOSwiftUI.Common.Messages.Base;

namespace IOSwiftUI.Common;

public class BOMemberListResponseModel : ResponseModel
{
    public int Count { get; set; }
    public IList<BOMemberModel> MemberList { get; set; }

    public BOMemberListResponseModel(int count, IList<BOMemberModel> memberList) : base()
    {
        Count = count;
        MemberList = memberList;
    }
}
